### DISCUSSION TITLE
There's a buzzing/clicking noise when extreme amount of compression is applied

### DISCUSSION THREAD

**Discussion by acc4github** [ZL-Audio/ZLEqualizer#60] [Q&A] [answered]
It may be inevitable, but is there a way to reduce or prevent it with tweaking the options?

**Comment by zsliu98** [comment, marked as answer]
- you may add a limiter after the equalizer (if the peak becomes > 0 dB)
- you may increase the attack/release/smooth
- you may use the state variable structure instead
- you may use the high-quality option (I have just added this feature)
